operations/helm/charts/mimir-distributed/templates/podsecuritypolicy.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if eq (include "mimir.rbac.usePodSecurityPolicy" .) "true" }}
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```

Comment:
We don't even try to install PodSecutiryPolicy on 1.25 and above, due to https://github.com/grafana/mimir/blob/6ca0f9c033a53864194fd55a7744caf539628e1a/operations/helm/charts/mimir-distributed/templates/_helpers.tpl#L437